src/main/java/hu/oe/nik/szfmv/automatedcar/bus/packets/interfaces/IRadarPacket.java

```diff
@@ -0,0 +1,32 @@
+package hu.oe.nik.szfmv.automatedcar.bus.packets.interfaces;
+
+import hu.oe.nik.szfmv.environment.worldobjectclasses.Collidable;
+
+public interface IRadarPacket {
```

Comment:
Még annyi , hogy implementálni is kéne az interfacet mert így nem tudjuk példányosítani és akkor ott megadhatnád, hogy a saját sávja egy két elemű tömb! De amúgy maga az iterface részemről jó!
@kovacsistvan1

Reply:
@bex1111  kész van, majd nézzétek át kérlek.